`cargo run --release` got an error

**Describe the bug**
`cargo run --release` got an error.


**To Reproduce**
```
➜  arrow-datafusion git:(parking_lot) cargo run --release --bin tpch
   Compiling hyper-timeout v0.4.1
   Compiling datafusion v6.0.0 (/Users/bytedance/arrow-datafusion/datafusion)
   Compiling tonic v0.6.2
error[E0425]: cannot find function `is_sorted` in this scope
   --> datafusion/src/physical_plan/tdigest/mod.rs:270:27
    |
270 |             debug_assert!(is_sorted(sorted_values), "unsorted input to TDigest");
    |                           ^^^^^^^^^ not found in this scope

   Compiling arrow-flight v8.0.0
```

**Expected behavior**
No error

**Additional context**
Add any other context about the problem here.

--- a/datafusion/src/physical_plan/tdigest/mod.rs
+++ b/datafusion/src/physical_plan/tdigest/mod.rs
@@ -266,6 +266,7 @@ impl TDigest {
     }
 
     fn merge_sorted_f64(&self, sorted_values: &[OrderedFloat<f64>]) -> TDigest {
+        #[cfg(debug_assertions)]
         debug_assert!(is_sorted(sorted_values), "unsorted input to TDigest");
 
         if sorted_values.is_empty() {

